web-sdk/README.md

```diff
@@ -20,6 +20,9 @@ You can easily integrate the Web SDK in your website or mobile app with only a f
 
 *Note: You can still use v1, by setting the script `src` to `https://resources.fidel.uk/sdk/js/v1/fidel.js`.*
 
+
+Here are the parameters that must be defined in your JavaScript.  All of the parameters that begin with `data-` can be customised for your application, but `data-program-id`, `data-key`, `data-country-code` are the absolute minimum to have reports arrive back your Fidel Dashboard.  The full parameter description is found below the code.
```

Comment:
Some improvement suggestions here: 

`have reports arrive back your Fidel Dashboard` -> `successfully link cards to your program`

`The full parameter description is found` -> `You can find the full list of parameters`

Reply:
please address this one as well and remove the extra spaces between sentences